EpiCommonUtils/Infrastructure/EditorDescriptors/UseUnderlyingTypeAttribute.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Forte.EpiCommonUtils.Infrastructure.EditorDescriptors
+{
+    /// <summary>
+    /// Use it on enum property to force EnumSelectionFactory to send its value to Episerver UI using underlying type.
+    /// For example, it might be useful to bypass custom JsonConverter applied to the enum.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property)]
```

Comment:
I don't have strong opinion here but I feel it would be better to use this annotation on Enum level rather than property level. My rationale behind this is that the reason you want to use this attribute in the first place is that you changed default json converter on to StringEnumConverter. You did it on Enum level so maybe it's better to do these two related actions on the same level?

Reply:
I think that it would be less general then. In our (specific) use case it could work well. But let's imagine that we want to use enum type which is defined in an external package and we cannot alter its definition (well, it's a rather rare scenario, but not impossible) - then you have only control over defined property.